src/ephemera/powerups.js

```diff
@@ -0,0 +1,57 @@
+// import Phaser from 'phaser'
+import {curry} from 'ramda'
+import {CONFIG, GAME_CONFIG} from '../config'
+import {WORLD} from '../world'
+import {Circle, throtLog} from '../util'
+
+// const {Geom} = Phaser
+// const {Circle} = Geom
+
+const {powerups} = CONFIG
+/*
+ */
+
+export const addPowerUp = curry((which, creator) => () => {
+  let id
+  const base = powerups[which]
+  // console.log(`generating powerup maker for ${base.sprite}`)
+  const render = (entity) => {
+    // console.log(`adding powerup ${base.sprite}`)
+    // const {entity} = creator(base)
+    if (!entity) {
+      entity = creator(base)
+      WORLD.ephemera.powerups.push(entity)
+    }
+    WORLD.graphics.lineStyle(2, 0xffff00, 2)
+    WORLD.graphics.fillStyle(0xff0000, 1)
+    WORLD.graphics.fillCircle(entity.x, entity.y, entity._radius)
+    // console.log(WORLD.graphics, `world gfx`)
+  }
```

Comment:
I'd like for this to be less hardcoded over time, but in my testing it was a PITA to set up, so I figured layering it in later would be ok.